lib/charms/kubernetes_charm_libraries/v0/multus.py

```diff
@@ -543,6 +543,23 @@ def _container_security_context_is_set(
                     return False
         return True
 
+    def multus_is_available(self) -> bool:
+        """Check whether Multus is enabled leveraging existence of NAD custom resource.
+
+        Returns:
+            bool: Whether Multus is enabled
+        """
+        try:
+            list(self.client.list(res=NetworkAttachmentDefinition, namespace=self.namespace))
+        except httpx.HTTPStatusError as e:
+            if e.response.status_code == 404:
+                return False
+            else:
```

Comment:
We can also add a test case for the else clause (returns error code other than 404).